common/lib/xmodule/xmodule/video_module/video_handlers.py

```diff
@@ -383,6 +383,24 @@ def transcript(self, request, dispatch):
 
         return response
 
+    @XBlock.handler
+    def student_view_user_state(self, request, suffix=''):
+        """
+        Endpoint to get student view data JSON.
+
+        Retrieve user-specific information from video xblock.
```

Comment:
Nit: I think the docstring would be more clear if you changed the whole thing (both lines) to:

> Endpoint to get user-specific state, like current position and playback speed, without rendering the full student_view HTML. This is similar to student_view_state, but that one cannot contain user-specific info.

Reply:
Thanks for the suggestion! Updated :)